<launch>
  
  <!-- <arg name="map" default="octomap_full.bt"/> -->
  <arg name="global_frame_id" default="map"/>

  <arg name="map_path" default="/home/simon/marsupial_ws/src/marsupial_optimizer/maps/thermal_central.bt"/>


  <node name="grid3d_node" type="grid3d_node" pkg="mcl3d" output="screen" args="$(arg map_path)">
    <!-- <param name="map_path" value="$(find marsupial_optimizer)/maps/$(arg map)" /> -->
    <param name="global_frame_id" value="$(arg global_frame_id)" />
    <param name="publish_point_cloud" value="true" />
    <param name="publish_grid_slice" value="0.0"/>
    <param name="sensor_dev" value="0.02" />  
  </node>

</launch>	
